slack/resource_conversation.go

```diff
@@ -246,12 +264,19 @@ func resourceSlackConversationDelete(ctx context.Context, d *schema.ResourceData
 	client := m.(*slack.Client)
 
 	id := d.Id()
-	err := archiveConversationWithContext(ctx, client, id)
-	if err != nil {
-		if err.Error() == "channel_not_found" {
-			return diags
+	action := d.Get("action_on_destroy").(string)
+	switch action {
+
+	case conversationActionOnDestroyNone:
+		fmt.Sprintf("Do nothing on Conversation: %s (%s)", id, d.Get("name"))
```

Comment:
This can be replaced with 
```go
	case conversationActionOnDestroyNone:
		diags = append(diags, diag.Diagnostic{
			Severity: diag.Warning,
			Summary:  fmt.Sprintf("conversation %s (%s) won't be archived on destroy", id, d.Get("name")),
			Detail:   fmt.Sprintf("action_on_destroy is set to %s which does not archive the conversation ", conversationActionOnDestroyNone),
		})
```

or something similar

Reply:
fixed